src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1748,6 +1748,10 @@ def _get_public_ip_address_allocation(value, sku):
 
     # Handle load balancer creation
     if load_balancer_type == 'new':
+        # Defaults SKU to 'Standard' for zonal scale set
+        if load_balancer_sku is None:
+            load_balancer_sku = 'Standard' if zones else 'Basic'
```

Comment:
Ah, the answer to my earlier question.  We need to call this behavior out in the help.